lib/matplotlib/backends/backend_gtk.py

```diff
@@ -613,6 +613,8 @@ def destroy(self, *args):
     def show(self):
         # show the figure window
         self.window.show()
+        self.window.set_keep_above(True)
```

Comment:
It's probably useful to document with a comment that this is intended so that no one "optimises" it away